.github/workflows/happy-path.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# che-plugin-registry Happy Path workflow
+# matrix jobs with alpine and ubi8
+name: Happy Path
+
+on: [push, pull_request]
```

Comment:
This will run for sidecar update PRs, which I don't think is testing anything because the generated `meta.yaml` files won't be referencing it. Maybe it should only run on updates to `che*.yaml` files?

Reply:
If you change a sh script or Dockerfile or `build` library, etc it requires as well to be tested
Maybe everything except `sidecars` folder ?

Reply:
Yes sounds good :+1: